### ISSUE TITLE
WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (.inactive)

### ISSUE THREAD

**Issue by BorisGuan** [BorisGuan/MDSIssuesRepro#3]
#### Issue
`Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds` ([color-contrast](https://accessibilityinsights.io/info-examples/web/color-contrast))
#### Target application
[Explore - Master Data Services](http://eims-mds-01/MDS/Explore/Entity?MID=491428b6-219a-4e75-940d-68461ca0e0b0&VID=da824cc4-89d2-45cf-a999-215411e2de2d&EID=84b9753c-527a-4a1e-a486-f115dd47aaa1)
#### Element path
.inactive
#### Snippet
`<a role="button" href="#" data-bind="visible:!isViewHistory(), click:MemberAnnotationsPage.HasHistory()? MemberAnnotationsPage.ShowMemberAnnotationsCommand : null, css: {inactive: !MemberAnnotationsPage.HasHistory()}" class="inactive">View History</a>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 3.94 (foreground color: #808080, background color: #ffffff, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 88.0.4324.104
This accessibility issue was found using Accessibility Insights for Web 2.23.0 (axe-core 4.1.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
